game/gbtd_gbr0.ksy

```diff
@@ -0,0 +1,217 @@
+meta:
+  id: gbr0
+  endian: le
+  encoding: ascii
+  title: GBR file
+  license: MIT
+  application: GameBoy Tile Designer, GBTD
+  file-extension:
+    - gbr
+  tags:
+    - game
+
+doc: |
+  GBR0 files are used by GameBoy Tile Designer
+  to store tilemaps created with the program.
+doc-ref: http://www.devrs.com/gb/hmgd/gbtd.html
```

Comment:
This website is available via TLS.

Reply:
Also https://github.com/bashaus/gbtiles

Reply:
Fixed https but I don't think that gbtiles repo adds any extra value.